src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/HalfConverter.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+
+namespace System.ComponentModel
+{
+    /// <summary>
+    /// Provides a type converter to convert half-precision, floating point number objects to and
+    /// from various other representations.
+    /// </summary>
+    public class HalfConverter : BaseNumberConverter
+    {
+        /// <summary>
+        /// Determines whether this editor will attempt to convert hex (0x or #) strings
+        /// </summary>
+        internal override bool AllowHex => false;
+
+        /// <summary>
+        /// The Type this converter is targeting (e.g. Int16, UInt32, etc.)
+        /// </summary>
+        internal override Type TargetType => typeof(Half);
+
+        /// <summary>
+        /// Convert the given value to a string using the given radix
+        /// </summary>
+        internal override object FromString(string value, int radix) => value is null ? (object)0 : Half.Parse(value, CultureInfo.CurrentCulture);
```

Comment:
Similar comment here about the Single and Double converters calling `Convert.ToSingle/Double`. Do we not have an equivalent Half method?